src/main/java/org/opentripplanner/middleware/controllers/api/AbstractUserController.java

```diff
@@ -101,8 +102,17 @@ private Job resendVerificationEmail(Request req, Response res) {
      */
     @Override
     U preCreateHook(U user, Request req) {
-        User auth0UserProfile = createNewAuth0User(user, req, this.persistence);
-        return updateAuthFieldsForUser(user, auth0UserProfile);
+        RequestingUser requestingUser = Auth0Connection.getUserFromRequest(req);
+        // TODO: If MOD UI is to be an ApiUser, we may want to do an additional check here to determine if this is a
+        //  first-party API user (MOD UI) or third party.
+        if (requestingUser.apiUser != null && user instanceof OtpUser) {
+            // Do not create Auth0 account for OtpUsers created on behalf of third party API users.
+            return user;
```

Comment:
Does `applicationId` need to be applied here or does that happen elsewhere?

Reply:
@landonreed This happens here: OtpUserController -> preCreateHook. But I'm not sure this is correct. As it stands this would mean only Api users being able to create Otp users which I don't think is correct.

Reply:
I've updated.